static/app/views/settings/organizationAuth/providerItem.tsx

```diff
@@ -79,6 +79,16 @@ const ProviderItem = ({provider, active, onConfigure}: Props) => {
 
   const featureProps = hookName ? {hookName} : {};
 
+  function GetProviderDescription(p: string) {
```

Comment:
two nits: 
- I think we generally use arrow function declarations instead of the `function` keyword. 
- we generally don't use one letter variable names except for css styling on the frontend at least.